pageserver/src/tenant/timeline.rs

```diff
@@ -4240,6 +4224,7 @@ impl Timeline {
                         img_range.start,
                         img_range.end
                     );
+                    start = img_range.end;
```

Comment:
Guessing this was a bug

Reply:
Do we know how bad the impact can be? I see that we only touch this codepath when force compacting via the http api.

Reply:
yes, this only impacts tests now, I don't think anyone has run force compaction on prod yet.